atomicapp/nulecule/base.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, id, specversion, graph, basepath, metadata=None,
             params (list): List of params for the Nulecule application
             config (dict): Config data for the Nulecule application
             namespace (str): Namespace of the current Nulecule application
+            cli (dict): CLI data
```

Comment:
Ah woops, my recent comment is the same as what @dustymabe just said about this PR. I agree with him too. Should only be passing it in once.